boxnav/boxsim.py

```diff
@@ -8,6 +8,7 @@
 from boxnav.boxenv import BoxEnv
 from boxnav.boxnavigator import BoxNavigator, Navigator
 from boxnav.environments import oldenborg_boxes as boxes
+from time import sleep
```

Comment:
We probably want to remove this from the final version. Good for debugging, but no reason to keep it around once you have a working version.